process_pr.py

```diff
@@ -745,19 +747,20 @@ def check_test_cmd(first_line, repo, params):
         prs = []
         cmssw_que = ""
         logger.debug("check_test_cmd: %s", m.groups())
-        if m.group(6):
-            wfs = ",".join(set(m.group(6).replace(" ", "").split(",")))
-        if m.group(11):
-            prs = get_prs_list_from_string(m.group(11), repo)
-        if m.group(20):
-            cmssw_que = m.group(20)
-        if m.group(25):
-            if "addpkg" in m.group(25):
-                params["EXTRA_CMSSW_PACKAGES"] = m.group(27).strip()
+        build_only = m.group(4) == "build"
+        if m.group(7):
+            wfs = ",".join(set(m.group(7).replace(" ", "").split(",")))
+        if m.group(12):
+            prs = get_prs_list_from_string(m.group(12), repo)
+        if m.group(21):
+            cmssw_que = m.group(21)
+        if m.group(26):
+            if "addpkg" in m.group(26):
+                params["EXTRA_CMSSW_PACKAGES"] = m.group(28).strip()
             else:
                 params["BUILD_FULL_CMSSW"] = "true"
-        return (True, " ".join(prs), wfs, cmssw_que)
-    return (False, "", "", "")
+        return (True, " ".join(prs), wfs, cmssw_que, build_only)
+    return (False, "", "", "", "")
```

Comment:
@iarspider , though works but batter to return False i.e. `eturn (False, "", "", "", False)`